supportedBackends/sfs/src/main/scala/cromwell/backend/sfs/SharedFileSystemAsyncJobExecutionActor.scala

```diff
@@ -167,10 +167,25 @@ trait SharedFileSystemAsyncJobExecutionActor
     * Writes the script file containing the user's command from the WDL as well
     * as some extra shell code for monitoring jobs
     */
-  private def writeScript(instantiatedCommand: String, cwd: Path) = {
+  private def writeScript(instantiatedCommand: String, cwd: Path, globFiles: Set[WdlGlobFile]) = {
     val rcPath = if (isDockerRun) jobPaths.toDockerPath(jobPaths.returnCode) else jobPaths.returnCode
     val rcTmpPath = s"$rcPath.tmp"
 
+    def globManipulation(globFile: WdlGlobFile) = {
```

Comment:
True -- we had in person discussion that much more can/should be abstracted, but I was going to file a future ticket for it, in the interest of time. However, I'll see what I can tweak once I get the test to pass.